api/lib/infrastructure/mailers/mailer.js

```diff
@@ -22,6 +22,10 @@ class Mailer {
   }
 
   async sendEmail(options) {
+    if (!mailing.enabled) {
+      return EmailingAttempt.success(options.to);
+    }
+
     debugEmail(options);
```

Comment:
Laisser actif le debug pour l'envoi d'email, même si `MAILING_ENABLED=false`, introduit avec #5577 : 

```suggestion
    debugEmail(options);

    if (!mailing.enabled) {
      return EmailingAttempt.success(options.to);
    }
```